packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -78,3 +80,81 @@ export interface SetStringOptions {
    */
   inputFormat?: StringFormat;
 }
+
+// @needsAudit
+export interface ApplePasteButtonProps extends ViewProps {
+  /**
+   * A callback that is called with the result of the paste action.
+   * Inspect the `type` property to determine the type of the pasted data.
+   
+   * Can be one of `text` or `image`
+   *
+   * @example
+   * ```ts
+   *   onPress={(data) => {
+   *     if (data.type === 'image') {
+   *       setImageData(data);
+   *    } else {
+   *       setTextData(data);
+   *     }
+   *   }}
+   * ```
+   */
+  onPress: (data: PasteEventPayload) => void;
+  /**
+   * The backgroundColor of the button.
+   */
+  backgroundColor?: string;
+  /**
+   * The foregroundColor of the button.
+   */
+  foregroundColor?: string;
+  /**
+   * The cornerStyle of the button.
+   * @default capsule
+   *
+   * @see [Apple
+   * Documetation](https://developer.apple.com/documentation/uikit/uibutton/configuration/cornerstyle) for more details.
+   */
+  cornerStyle?: CornerStyle;
+  /**
+   * The displayMode of the button.
+   * @default `iconAndLabel`
+   *
+   * @see [Apple
+   * Documetation](https://developer.apple.com/documentation/uikit/uipastecontrol/displaymode) for more details.
+   */
+  displayMode?: DisplayMode;
+  /**
+   * The custom style to apply to the button. Should not include `backgroundColor`, `borderRadius` or `color`
+   * properties.
```

Comment:
I guess Apple doesn't let us to change these styles, right? If so, can we link to their docs/guidelines?

Reply:
Yes, this nothing here works on the button. This is just for the regular `style` object. We link to the docs for `displayMode` and `cornerStyle`above

Reply:
I mean more something like we already have for the authentication button, where we clearly say that this is against Apple's guidelines, see https://docs.expo.dev/versions/latest/sdk/apple-authentication/#appleauthenticationbutton